app/models/cart.rb

```diff
@@ -11,10 +12,9 @@ def add_item(item_id)
   end
 
   def minus_item(item_id)
-    if contents[item_id.to_s] > 0
+    if contents[item_id.to_s] >= 1
       contents[item_id.to_s] -= 1
-    else contents[item_id.to_s] == 0
-      contents.delete_if {|key,value| value == contents[item_id.to_s]}
+      contents.keep_if { |key,value| value >= 1 }
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.<br>Space missing after comma.